.github/workflows/AssetsUpdater.yml

```diff
@@ -0,0 +1,37 @@
+name: Assets Updater
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.ts"
+      - "**.json"
+jobs:
+  update:
+    if: github.repository_owner == 'PreMiD'
```

Comment:
Don't think this is needed because Workflows are disabled by default on forks now

Reply:
Copied from other workflows we have, we should change at a later point